src/ReactWrapper.js

```diff
@@ -269,8 +270,12 @@ export default class ReactWrapper {
    * @returns {ReactWrapper}
    */
   find(selector) {
-    const predicate = buildInstPredicate(selector);
-    return findWhereUnwrapped(this, predicate);
+    const complexSelector = new ComplexSelector(
+      buildInstPredicate,
+      findWhereUnwrapped,
+      childrenOfInst
+    );
```

Comment:
Originally I was currying it but it didn't seem as clean.

```js
complexSelector(buildInstPredicate, findWhereUnwrapped, childrenOfInst)(selector, this);
```

I guess it could all be one function, but I wanted to separate initialization from the logic.